include/mconfig.h

```diff
@@ -185,7 +187,7 @@ struct MConfig {
 	struct MConfigSwipeCoasting{
 		float min_speed;		// What speed to start scroll coasting at. >= 0
 		int tick_ms;		// How fast events will be generated during coasting >= 1
-		int duration;		// How long coasting ticks will last >= 0, 0 disables coasting
+		float duration;		// How long coasting ticks will last >= 0, 0 disables coasting
```

Comment:
This should stay as int. Readme.md says it's real number so it should be corrected too.

Reply:
Traces back to some commit about removing all warnings. There was likely a float to int truncation error related to this, but I'll try to change it back to int and fixing the truncation error elsewhere.